File: Array Searching Algorithms/ArrayLinearSearch.js
let arr = [20, 45, 27, 47, 55, 67, 75, 88, 90];
function find(arr, n) {
  for (var i = 0; i < arr.length; i++) {
    if (arr[i] == n) {
      return i;
      break;
    }
  }
  return -1;
}
console.log(find(arr, 67));

/* 
Psudo code
 arr =[array contents]
x = arr.length
for i in range(0,x){
    if(arr[i] == n ) {
        return i;
        break;
    }
}
return -1;
*/

File: Array Searching Algorithms/ArrayLinearSearch.py
##Function Code
##time Complexity: O(n)
##space complexith: O(1)
def linearSearch(arr,x): 
    for i in range(len(arr)):
        if arr[i] == x:
            return i
    return -1


## Driver Code
arr = [20,45,27,47,55,67,75,88,90]
x = 67
result = linearSearch(arr, x)
print(result)
